### PR TITLE
docs(aio): add ant design of angular in resources

### PR DESCRIPTION
**Pull request by vthinkxie**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by vthinkxie** [comment]
Hi @StephenFluin 
we just finish the english version of our document
here is our [github](https://github.com/NG-ZORRO/ng-zorro-antd) and [doc site](https://ng.ant.design/docs/introduce/en)

**Review by StephenFluin — APPROVED** [review]

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -345,6 +345,12 @@
             "title": "Angular Material",
             "url": "https://github.com/angular/material2"
           },
+          "ngzorro": {
+            "desc": "An enterprise-class UI components based on Ant Design and Angular",
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: _An enterprise-class UI components_ --> _A set of enterprise-class UI components_ (?)

File: aio/content/marketing/resources.json [hunk 2]
```diff
@@ -345,6 +345,12 @@
...
+            "rev": true,
+            "title": "Ant Design of Angular(ng-zorro-antd)",
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: Shouldn't there be a space before `(`? :grin:

**Comment by vthinkxie** [comment]
Hi @gkalpak , thanks for your suggestion, I have updated the description.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Awesome! Can you please squash the commits into one?

**Comment by vthinkxie** [comment]
@gkalpak done

**Pull request closed without merging**